docs/zh_cn/index.rst

```diff
@@ -15,6 +15,7 @@
    :caption: 教程
 
    tutorials/chapter_01_introduction_to_model_deployment.md
+   tutorials/chapter_02_problems.md
```

Comment:
How about renaming "problems" to "chanllenges"?

Reply:
> How about renaming "problems" to "chanllenges"?

Sure, done